frontend/src/certificate_form.tsx

```diff
@@ -0,0 +1,245 @@
+                  value={individual.birth_date ?? ""}
+                  InputLabelProps={{
+                    shrink: true,
+                  }}
+                  onChange={(date, value) => {
+                    value && onUpdateIndividual("birth_date", value);
+                  }}
+                />
+              </div>
+              <div className={style.flexRow}>
+                <Autocomplete
+                  disabled={!canManage}
+                  options={sexOptions ?? []}
+                  value={
+                    sexOptions.find(
+                      (option) => option.value == individual.sex
+                    ) ?? sexOptions[sexOptions.length - 1]
+                  }
+                  getOptionLabel={(option: OptionType) => option.label}
+                  renderInput={(params) => (
+                    <TextField
+                      {...params}
+                      label="Kön"
+                      className={style.control}
+                      variant={inputVariant}
+                      margin="normal"
+                    />
+                  )}
+                  onChange={(event: any, newValue: OptionType | null) => {
+                    onUpdateIndividual("sex", newValue?.value ?? "");
+                  }}
+                />
+                <TextField
+                  disabled={!canManage}
+                  label="Antal födda i kullen"
+                  className={style.control}
+                  variant={inputVariant}
+                  value={individual.litter ?? null}
+                  type="number"
+                  onChange={(event) => {
+                    onUpdateIndividual("litter", +event.currentTarget.value);
+                  }}
+                />
+              </div>
+              <div className={style.flexRow}>
+                <Autocomplete
+                  disabled={!canManage}
+                  options={colorOptions ?? []}
+                  value={
+                    colorOptions.find(
+                      (option) => option.value == individual.color
+                    ) ?? colorOptions[0]
+                  }
+                  getOptionLabel={(option: OptionType) => option.label}
+                  renderInput={(params) => (
+                    <TextField
+                      {...params}
+                      label="Färg"
+                      className={style.control}
+                      variant={inputVariant}
+                      margin="normal"
+                    />
+                  )}
+                  onChange={(event: any, newValue: OptionType | null) => {
+                    onUpdateIndividual("color", newValue?.value ?? "");
+                  }}
+                />
+                <TextField
+                  disabled={!canManage}
+                  label="Avvikande hårlag"
+                  variant={inputVariant}
+                  className={style.control}
+                  multiline
+                  rows={1}
```

Comment:
Is this something prettier does? It does look kind of odd to me, but if prettier thinks that's the way, I'm fine with it.